theoads/getting-started/00-getting-started-web.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 0.2
+sidebar_label: Web
+---
+
+# Web
+
+Setting up THEOads requires some small changes to your THEOplayer build, as well as setting up a signaling server.
+
+:::note
+Currently only HLS streams are supported. In the future DASH and HESP streams will also be supported.
+:::
+
+## Prerequisites
+
+### 1. Your THEOplayer SDK needs to have the 'theoads' feature enabled.
+
+The THEOads feature is only included in the `@theoplayer/theoads` package which can be installed by executing the following command:
+
+```bash
+npm install @theoplayer/theoads
```

Comment:
We should start recommending people to install their chosen variant with an `theoplayer` alias, like we do for the [Web UI](https://www.theoplayer.com/docs/open-video-ui/web/getting-started/).
```suggestion
npm install theoplayer@npm:@theoplayer/theoads
```